pycontw/pycon.tw
# Keynote view api

## Types of changes


## Description

Add KeynoteEventListAPIView and update KeynoteEvent model

    - speaker: contains speaker name, bio, and photo
    - session: contains keynote title, description, and slides
    - social: links of speaker's social account,  contains Linkedin. Twitter, Github
    - slido: slido link for the keynote
    

## Steps to Test This Pull Request
1. Log in to 'http://127.0.0.1:8000/admin'
2. Add test data to events/Keynote
3. Check 'http://127.0.0.1:8000/api/events/keynotes/' for outcome

## Outcome
<img width="1254" alt="截圖 2021-06-27 下午6 28 42" src="https://user-images.githubusercontent.com/13831865/123541156-907a1f00-d775-11eb-9d23-facd46a4fcf6.png">
